ios/FluentUI/Table View/TableViewTokens.swift

```diff
@@ -149,3 +155,16 @@ open class TableViewCellTokens: ControlTokens {
     case small
     case medium
 }
+
+open class ActionsCellTokens: TableViewCellTokens {
+    /// The regular text color in an ActionsCell
+    open var regularTextColor: DynamicColor { globalTokens.brandColors[.primary] }
+
+    /// The destructive text color in an ActionsCell.
+    open var destructiveTextColor: DynamicColor { DynamicColor(light: ColorValue(0xD92C2C),
```

Comment:
we can't use the actual danger color in the fluent code? these are things then @edjamesmsft  needs to update later?

Reply:
Do we have a danger color token for Fluent 2?

Reply:
No, because the tokens need to be of DynamicColor and we currently don't have an established danger color in new fluent tokens.